### DISCUSSION TITLE
BOOK LIB CONNECT - BOOKS NO LONGER IN LIBRARY<title>

### DISCUSSION THREAD

**Discussion by mikel1954** [community/community#77363] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have used AAX Audio Converter for several years with no problems, and Book Lib Connect since it was released, likewise.  I find both programmes really easy to use.  However, earlier today I tried to download a book in my Audible Library using BLC, but when I clicked the “download” button, the book was not there (although other books which were no longer in my library were shown).  I clicked on “Resync” but this made no difference. 
I deleted the profile and then created it again (and also deregistered the associated device on my amazon account), but now no books are being shown at all!   
I therefore uninstalled BLC and reinstalled it, but it hasn’t made any difference; none of the books in my library are showing when I click on “download”.  I have read through the User Guide, but can’t find anything to help me.   
I would be very grateful if someone could help.  Thank you.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by mikel1954** [reply]
Thanks but, as I said, I have read through the User Guide and there doesn't seem to be anything there to help.  Moreover, I have used Book Lib Connect successfully since it was first released, so I don't understand why I am suddenly having problems with it.

**Comment by gerlandotermini** [comment]
I'm having the same issue and can't figure out why it doesn't work anymore.

**Comment by CorroPenny** [comment]
I'm having the same issue. It just started having the issue a month or two ago. I've been using it for a while and have plents of books downloaded (from two different accounts). But now it's only showing books I already had, and not pulling my new ones.

**Comment by gvar1146** [comment]
Hey guys, has there been a solution for this yet? I just opened it today after buying a new audiobook and it said that my license was no longer valid. So after logging in again, no more library...

**Comment by mihilion** [comment]
I had the same issue, and deleting the database C:\Users\<your user name>\AppData\Local\audiamus\BookLibConnect\data\audiobooks.db helped me, as suggested by the author of the software: https://github.com/audiamus/BookLibConnect/issues/185#issuecomment-2701786762

**Reply by ghost** [reply]
This is the first time I've tried to set up a profile to connect to Amazon Audible. The Create Log-in Firefox URL launches the Amazon Audible and Amazon Company log-in page. However, when I enter the email user and password for the account and the puzzle, it does not work.

I keep getting the message, "Looking for something?  We're sorry.  The Web address you entered is not a functioning page on our site"

I can log in not issue directly. I wasted a lot of time connecting my audible library to the AAX Audio Converter.  What is wrong?

Why won't it connect?

**Reply by dyscodyva** [reply]
You have to copy the URL from that error message page and paste it into BookLib. If you can't get it to work, cancel the install and then delete C:\Users<your user name>\AppData\Local\audiamus\BookLibConnect\data\audiobooks.db and retry the install (using the instructions from the first part of my message). That worked for me as a first-time user.
